src/lib/parse/LibFlareFtsoSubParser.sol

```diff
@@ -24,6 +26,8 @@ library LibFlareFtsoSubParser {
             "ftso-current-price-pair",
             "Returns the current price of the given token pair according to the FTSO. Accepts 3 inputs, the symbol string used by the FTSO for the base token, the symbol string used by the FTSO for the quote token and the timeout in seconds. The price is rounded down if it does not fit in a Rainlang number. The timeout will be used to determine if the price is stale and revert if it is. Note that the pair price is derived from two separate FTSO prices mechanically and is not provided directly by the FTSO."
         );
+        meta[SUB_PARSER_WORD_SFLR_EXCHANGE_RATE] =
+            AuthoringMetaV2("sflr-exchange-rate", "Returns the current exchange rate of FLR to SFLR.");
```

Comment:
"exchange" feels a bit vague to me, maybe it should be "underlying" or something that indicates it's the real FLR backing per sFLR?

or maybe we should just copy their naming?

Reply:
apparently they call it exchange in their website but not  the contracts so we can stick with exchange